### PR TITLE
Footer Section 

### PR DESCRIPTION
**Pull request by sierra-acy**
- Added footer section
- Added mailto to email in footer (just like for Discord button)
- Made divider styling on header "tailwindy" (using tailwind styles instead of explicit style tag and regular css)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -89,6 +89,14 @@ <h2 class="card-title">Placeholder Title!</h2>
             </div>
         </div>
     </div>
+    <div class="divider mb-0 h-0"></div>
+    <!-- Footer -->
+    <footer class="footer footer-center bg-base-200 text-base-content p-4">
+        <aside>
+            <p class="text-wrap w-2/3 text-balance mb-3">For any questions, or to request an invite to the official Tea Game Discord server, please reach out to me at <a class="mailtoui link" href="mailto:teatimedeveloper@gmail.com?subject=Tea Time Discord Access&body=Hello! I would like to request a link to join the Tea Time discord server.">teatimedeveloper@gmail.com</a></p>
+            <p>© 2024 Danielle Purdes. All right reserved.</p>
```
Comment (danipurdes) [inline src/index.html]: Nit: All right**s** reserved

File: src/index.html [hunk 2]
```diff
@@ -89,6 +89,14 @@ <h2 class="card-title">Placeholder Title!</h2>
             </div>
         </div>
     </div>
+    <div class="divider mb-0 h-0"></div>
+    <!-- Footer -->
+    <footer class="footer footer-center bg-base-200 text-base-content p-4">
+        <aside>
+            <p class="text-wrap w-2/3 text-balance mb-3">For any questions, or to request an invite to the official Tea Game Discord server, please reach out to me at <a class="mailtoui link" href="mailto:teatimedeveloper@gmail.com?subject=Tea Time Discord Access&body=Hello! I would like to request a link to join the Tea Time discord server.">teatimedeveloper@gmail.com</a></p>
```
Comment (danipurdes) [inline src/index.html]: Do you want to link to your own portfolio or add your name as website developer?

**Review by danipurdes — COMMENTED** [review]
Just those two comments

File: src/index.html (hunk 1 above)
Comment (sierra-acy) [reply src/index.html]: Fixed!

File: src/index.html (hunk 2 above)
Comment (sierra-acy) [reply src/index.html]: Good thinking!

File: src/index.html [hunk 3]
```diff
@@ -94,9 +94,10 @@ <h2 class="card-title">Placeholder Title!</h2>
     <footer class="footer footer-center bg-base-200 text-base-content p-4">
         <aside>
             <p class="text-wrap w-2/3 text-balance mb-3">For any questions, or to request an invite to the official Tea Game Discord server, please reach out to me at <a class="mailtoui link" href="mailto:teatimedeveloper@gmail.com?subject=Tea Time Discord Access&body=Hello! I would like to request a link to join the Tea Time discord server.">teatimedeveloper@gmail.com</a></p>
-            <p>© 2024 Danielle Purdes. All right reserved.</p>
+            <p>© 2024 Danielle Purdes. All rights reserved.</p>
+            <p class="text-wrap w-2/3 text-balance mt-3 text-sm">Website by <a href="sierraacy.com" class="link">Sierra Acy</a></p>
```
Comment (danipurdes) [inline src/index.html]: Your link won't work unless you include http or https, it's a quirk of Github Pages, apparently. https://stackoverflow.com/questions/61970559/is-it-possible-to-add-a-hyper-link-to-a-github-pages

**Review by danipurdes — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 3 above)
Comment (sierra-acy) [reply src/index.html]: Good catch! Should be fixed now.

**Review by danipurdes — APPROVED** [review]

**Pull request merged**